cypress/integration/inventory/quick-marc.spec.js

```diff
@@ -1,159 +1,131 @@
+          });
         });
+      });
     });
 
-    it('should show type select', () => {
-      cy.get('[class*="quickMarcEditorRow--"]:last-child').contains('Type');
-    });
-
-    describe('adding some data to new 006 field and saving', () => {
-      before(() => {
-        cy.get('[class*="quickMarcEditorRow--"]:last-child select').select('c');
-
-        cy.get('[class*="quickMarcEditorRow--"]:last-child input[name*="Comp"]').type('jz');
-        cy.get('[class*="quickMarcEditorRow--"]:last-child input[name*="FMus"]').type('a');
-        cy.get('[class*="quickMarcEditorRow--"]:last-child input[name*="Part"]').type('d');
-        cy.get('[class*="quickMarcEditorRow--"]:last-child input[name*="Audn"]').type('g');
-        cy.get('[class*="quickMarcEditorRow--"]:last-child input[name*="Form"]').type('s');
-        cy.get('[class*="quickMarcEditorRow--"]:last-child input[name*="AccM[0]"]').type('|');
-        cy.get('[class*="quickMarcEditorRow--"]:last-child input[name*="LTxt[0]"]').type('|');
-        cy.get('[class*="quickMarcEditorRow--"]:last-child input[name*="TrAr"]').type('|');
+    describe('editing the 006/00 field', () => {
+      beforeEach('and adding a 006/00 field', () => {
+        // FIXME: I can't open edit page because of disabled edit button to rewrite by using interactors
+        cy.get('.quickMarcEditorAddField:last').click();
 
-        cy.do(Button('Save & close').click());
+        cy.get('[class*="quickMarcEditorRow--"]:last-child').find('input')
+          .then(([tag]) => {
+            cy.wrap(tag).type('006');
+          });
+      });
 
-        cy.get('#pane-instancedetails [class*="actionMenuToggle--"]', { timeout: 10000 }).click();
-        cy.get('#clickable-view-source', { timeout: 10000 }).click();
+      it('should show type select', () => {
+        cy.get('[class*="quickMarcEditorRow--"]:last-child').contains('Type');
       });
 
-      it('should show updated data in View MARC record', () => {
-        cy.contains('cjzadgs');
+      describe('adding some data to new 006 field and saving', () => {
+        beforeEach(() => {
+          cy.get('[class*="quickMarcEditorRow--"]:last-child select').select('c');
+
+          cy.get('[class*="quickMarcEditorRow--"]:last-child input[name*="Comp"]').type('jz');
+          cy.get('[class*="quickMarcEditorRow--"]:last-child input[name*="FMus"]').type('a');
+          cy.get('[class*="quickMarcEditorRow--"]:last-child input[name*="Part"]').type('d');
+          cy.get('[class*="quickMarcEditorRow--"]:last-child input[name*="Audn"]').type('g');
+          cy.get('[class*="quickMarcEditorRow--"]:last-child input[name*="Form"]').type('s');
+          cy.get('[class*="quickMarcEditorRow--"]:last-child input[name*="AccM[0]"]').type('|');
+          cy.get('[class*="quickMarcEditorRow--"]:last-child input[name*="LTxt[0]"]').type('|');
+          cy.get('[class*="quickMarcEditorRow--"]:last-child input[name*="TrAr"]').type('|');
```

Comment:
Is there something to do to make this work with interactors?

Reply:
Probably, but I cann't open that page to investigate what elements are used there. I wrote comment above

Reply:
👆 @ViktorSoroka07 or @BogdanDenis @NikitaSedyx @JohnC-80
Do you have any ideas on how to open MARC record page?

Reply:
@wKich, Maybe try to use another query since there is no result for this one?

Reply:
@ViktorSoroka07, what exactly?

Reply:
I don't know, what env do use?